DemoApp/android/app/build.gradle

```diff
@@ -65,12 +65,6 @@ import com.android.build.OutputFile
  * ]
  */
 
-// TODO Try removing this ext override when RN 0.57 released (https://github.com/facebook/react-native/issues/16906).
```

Comment:
Did you test in release build type for android?

Reply:
Yes I did. App Center Build works in release build and tab icons look fine.